infra/app/app-config/env-config/environment-variables.tf

```diff
@@ -19,11 +19,15 @@ locals {
   #   }
   # }
   secrets = {
-    # Example generated secret
-    # RANDOM_SECRET = {
-    #   manage_method     = "generated"
-    #   secret_store_name = "/${var.app_name}-${var.environment}/random-secret"
-    # }
+    CHAINLIT_AUTH_SECRET = {
+      manage_method     = "generated"
+      secret_store_name = "/${var.app_name}-${var.environment}/CHAINLIT_AUTH_SECRET"
```

Comment:
Where is `CHAINLIT_AUTH_SECRET` used?

Reply:
Added a comment to the file explaining this (it's used by Chainlit to sign the authentication tokens)